views/make/index.js

```diff
@@ -73,6 +73,13 @@ module.exports = view.extend({
         goPreviousView: function (e) {
             global.history.back();
         },
+        onBackClick: function () {
+            if (this.mode !== 'edit') {
+                this.goPreviousView();
+            } else {
+                this.page('/profile');
```

Comment:
Ah! good point. Can you just change this to the app admin details view instead? That would make more sense than the profile